packages/react-components/src/components/ControlBar.tsx

```diff
@@ -70,7 +70,14 @@ export const ControlBar = (props: ControlBarProps): JSX.Element => {
   }, [layout, styles?.root, theme]);
   return (
     <div className={mergeStyles(mainDivStyle)}>
-      <Stack className={controlBarClassName}>{props.children}</Stack>
+      <Stack
+        className={mergeStyles(controlBarClassName, {
+          borderRadius: (theme as PartialTheme)?.effects?.roundedCorner6,
+          boxShadow: (theme as PartialTheme)?.effects?.elevation8
```

Comment:
Do we expect Contoso to set `themes.effects.roundedCorner6` to change corners for surfaces as documented by [FluentUI](https://github.com/microsoft/fluentui/blob/dab45a2afe609709735b9b6e604a1af40d50e809/packages/theme/src/types/IEffects.ts#L52) and similarly `themes.effects.elevation8` to change elevation for command tools